docs/src/references/c8y-configuration-management.md

```diff
@@ -64,8 +91,38 @@ $ ls -l /etc/tedge/operations/c8y/c8y_DownloadConfigFile
 -rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/c8y_DownloadConfigFile
 ```
 
+The configuration plugin can also act as configuration proxy for child-devices.
+For that to work for a child device named `$CHILD_DEVICE_ID`,
+two files must be added under `/etc/tedge/operations/c8y/$CHILD_DEVICE_ID`
+in order to declare the associated capabilities to Cumulocity.
+
+These two files can be created using the `c8y_configuration_plugin --init` option and providing child names:
+
+```shell
+$ sudo c8y_configuration_plugin --init child-1 child-2
+
+$ ls -l /etc/tedge/operations/c8y/child-1/c8y_UploadConfigFile
+-rw-r--r-- 1 tedge tedge 95 Mar 22 14:24 /etc/tedge/operations/c8y/child-1/c8y_UploadConfigFile
+  
+$ ls -l /etc/tedge/operations/c8y/child-1/c8y_DownloadConfigFile
+-rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/child-1/c8y_DownloadConfigFile
```

Comment:
I have a bit better proposal (because @initard got confused at his first look)

```suggestion
$ sudo c8y_configuration_plugin --init child-1 child-2

$ ls -l /etc/tedge/operations/c8y/child-1
-rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/child-1/c8y_DownloadConfigFile
-rw-r--r-- 1 tedge tedge 95 Mar 22 14:24 /etc/tedge/operations/c8y/child-1/c8y_UploadConfigFile
  
$ ls -l /etc/tedge/operations/c8y/child-2
-rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/child-2/c8y_DownloadConfigFile
-rw-r--r-- 1 tedge tedge 95 Mar 22 14:24 /etc/tedge/operations/c8y/child-2/c8y_UploadConfigFile
```

Reply:
Indeed better. I will fix it.

Reply:
Probably, also better to mention `/etc/tedge/operations/c8y/child-1` is owned by `tedge:tedge`.

Reply:
Done